### PR TITLE
feat: use own inquirer implementation

### PR DESCRIPTION
**Pull request by anshumanv**
**What kind of change does this PR introduce?**
chore

**Did you add tests for your changes?**
Yes

**If relevant, did you update the documentation?**
No

**Summary**
Uses inquirer for prompt, same as yeoman-env

**Does this PR introduce a breaking change?**
No

**Other information**

### CODE REVIEW

**Comment by anshumanv** [comment]
Don't know how it got introduced, only PR I can find is https://github.com/webpack/webpack-cli/pull/1362 which was not merged

**Comment by snitin315** [comment]
Here https://github.com/webpack/webpack-cli/pull/1237

**Comment by alexander-akait** [comment]
hm, what about don't use `migrate` and `inquirer` inside webpack-cli package and implement logic on own side - less deps, less problems, less size, also we use it only in one place https://github.com/webpack/webpack-cli/blob/master/packages/webpack-cli/lib/utils/prompt-installation.js

**Comment by anshumanv** [comment]
Yes can think of own inquirer 🤔

File: packages/webpack-cli/lib/utils/prompt.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+        input: process.stdin,
+        output: stdout,
+    });
+
+    return new Promise((resolve) => {
+        rl.question(message, function (answer) {
+            // close the stream
+            rl.close();
+            // resolve with the input response
+            if (['y', 'yes'].includes((answer || defaultResponse).toLowerCase())) resolve(true);
```
Comment (alexander-akait) [inline packages/webpack-cli/lib/utils/prompt.js:13]: Please prefer to use `if {} else {}`

File: packages/webpack-cli/lib/utils/prompt.js [hunk 2]
```diff
@@ -9,8 +9,11 @@ const prompt = ({ message, defaultResponse, stdout }) => {
         rl.question(message, function (answer) {
             // close the stream
             rl.close();
+            const response = (answer || defaultResponse).toLowerCase();
             // resolve with the input response
-            if (['y', 'yes'].includes((answer || defaultResponse).toLowerCase())) resolve(true);
+            if (response === 'y' || response === 'yes') {
+                resolve(true);
+            }
             resolve(false);
```
Comment (alexander-akait) [inline packages/webpack-cli/lib/utils/prompt.js:17]: Here logic problem, you are always run `resolve(false)`, that is why I always recommend use `if {} else {}` or `if () { return }`

Comment (anshumanv) [reply packages/webpack-cli/lib/utils/prompt.js]: yep correct, fixed now

**Review by alexander-akait — APPROVED** [review]
Good job

**Review by snitin315 — APPROVED** [review]
🚀

**Comment by alexander-akait** [comment]
Thanks

**Pull request merged**